### PR TITLE
feat(tabbar): add new type of Tabbar

### PR DESCRIPTION
**Pull request by ZengBeauty**
## Description

Tabbar add new type && fit more style.

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by ZengBeauty** [comment]
企业微信20210602-190138@2x

File: src/components/tabbar/radio-button/group.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ class Group extends React.PureComponent {
     defaultActiveIndex: PropTypes.number,
     gutter: PropTypes.number,
     onChange: PropTypes.func,
+    type: PropTypes.string,
```
Comment (ShinyLeee) [inline src/components/tabbar/radio-button/group.js:39]: 可枚举的 type

**Review by ShinyLeee — CHANGES_REQUESTED** [review]
补下 typing

**Comment by ZengBeauty** [comment]
typing 详见 https://github.com/tuya/DefinitelyTyped/commit/a6803228d812a5dc663358d435a8dc262f1355a7

**Review by ShinyLeee — APPROVED** [review]

**Comment by ShinyLeee** [comment]
@ZengBeauty 合并下 commit，处理下 commit message，没有大驼峰，检查报错

**Pull request closed without merging**